README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
 You can also [download the compiled and minified assets (CSS, JavaScript) from
 GitHub](docs/installation/installing-from-dist.md).
 
-## Browser support
+## Accessibility
 
-GOV.UK Frontend will allow you to build services that comply with the [guidance
-in the Service Manual][service-manual-browsers].
+The GOV.UK Design System team regularly tests GOV.UK Frontend with a range of users to make sure it’s as accessible as possible. You can read [the results of our most recent test for accessibility](https://design-system.service.gov.uk/accessibility/).
```

Comment:
'the results of our most recent test for accessibility' 

this wont always be an audit, I think we could give a more general statement that'd cover other things like internal reviews etc

Reply:
Ah good point. How about "You can read about [our most recent accessibility tests](https://design-system.service.gov.uk/accessibility/)."?